src/components/JumpIn/JumpIn.types.ts

```diff
@@ -1,4 +1,6 @@
 import { ChainId } from '@dcl/schemas'
+import { Dispatch } from 'redux'
```

Comment:
Would you mind moving this import one row up? 🙏

Reply:
Moved it!